app/src/androidTest/java/org/simple/clinic/user/UserSessionAndroidTest.kt

```diff
@@ -111,4 +112,31 @@ class UserSessionAndroidTest {
 
     assertThat(selectedCountryPreference.get()).isEqualTo(Optional.of(country))
   }
+
+  @Test
+  fun fetching_the_user_and_facility_details_should_work_as_expected() {
+    // given
+    val newFacility = TestData.facility(
+        uuid = UUID.fromString("56ddc7df-6a81-42bc-8659-78b0ccb51edd"),
+        syncGroup = "bb92c082-d749-478a-bd2a-fe7f10055ef0"
+    )
+    facilityRepository
+        .save(listOf(newFacility))
+        .blockingAwait()
+
+    facilityRepository
+        .setCurrentFacility(newFacility)
```

Comment:
`setCurrentFacility` is expecting facilityUuid here. Hence the test is failing.

Reply:
Fixing.